pep-0685.rst

```diff
@@ -95,10 +95,10 @@ Tools generating metadata MUST raise an error if a user specified
 two or more extra names which would normalize to the same name.
 Tools generating metadata MUST raise an error if an invalid extra
 name is provided as appropriate for the specified core metadata version.
-If an older core metadata version is specified and the name would be
+If a project's metadata specifies an older core metadata version and the name would be
 invalid with newer core metadata versions,
-tools SHOULD warn the user.
-Tools SHOULD warn users when an invalid extra name is read and SHOULD not use
+tools reading that metadata SHOULD warn the user.
+Tools SHOULD warn users when an invalid extra name is read and SHOULD ignore
 the name to avoid ambiguity.
```

Comment:
```suggestion
Tools SHOULD warn users when an invalid extra name is read and SHOULD ignore
the extra to avoid ambiguity.
```

Maybe I'm splitting hairs, but for clarity's sake, wouldn't it be the *extra* that is ignored, (i.e. not installed), not just the name used for it?

Reply:
I don't see there's a significant difference, but I'm not sufficiently bothered that I'll argue either way. Both "extra" and "name" clearly (IMO) refer back to the term "extra name".

Note that this sentence doesn't necessarily refer to installers - the tool might be an analysis utility that's reporting what extras are present in a set of projects, for example.